ui/public/locales/en.json

```diff
@@ -402,7 +402,7 @@
 "label.attaching": "Attaching",
 "label.authentication.method": "Authentication Method",
 "label.authentication.sshkey": "System SSH Key",
-"label.autofill.vcenter.credentials": "Autofill vCenter credentials",
+"label.autofill.vcenter.credentials.from.zone": "Autofill vCenter credentials from Zone",
```

Comment:
Would it make more sense (language-wise) something like    "Use existing vCenter details from the Zone" ?

Reply:
or ".... from this Zone"

Reply:
Yes, Use existing vCenter credentials from the Zone" is better.

'.... from the Zone' should be good enough as the Zone field is already present  in the Cluster create form.

Reply:
Done